test/test_helper.rb

```diff
@@ -5,11 +5,17 @@
 require 'ar_transaction_changes'
 require 'minitest/autorun'
 
+MiniTest::Test = MiniTest::Unit::TestCase unless defined?(MiniTest::Test)
```

Comment:
Isn't it now `Minitest::Test` ?  That is what you used as the base class for `TransactionChangesTest`.